### ISSUE TITLE
Bet button value and Raise button value located in two different areas. A distinct Bet button set up should be implemented

### ISSUE THREAD

**Issue by SirFrancis83** [dickreuter/Poker#174]
Hello,
in setting up a particular poker table refering to People's Poker I came across a problem with raise and bet button vaue locations. In this poker table the corrisponding values are located in two different areas. Therefore if you correctly set up the raise value Bot cannot recognize the bet value and viceversa. In addition this poker client does not show the all in value when bot must go all in to call.
I attach some screenshots that show it.

![2](https://user-images.githubusercontent.com/125397820/235984045-5ea95561-a192-4fdf-90dd-d73ea9336a1f.png)
![3](https://user-images.githubusercontent.com/125397820/235984053-d9cb36f0-e352-484f-951d-29604e294cbc.png)
![1](https://user-images.githubusercontent.com/125397820/235984063-247b5d21-00fd-44cb-9977-5be980dcbdb0.png)

**Comment by SirFrancis83** [comment]
So if I have understood what you are telling me, i should point as all in value the value of my funds? I can test and tell you if it works.

**Comment by dickreuter** [comment]
Yes. Try that.

**Comment by SirFrancis83** [comment]
Yes it seems to work the same👍

**Comment by dickreuter** [comment]
What about the raise button. Does it work if you mark the area to read the number to the whole button? It should automatically ignore any letters.

**Comment by SirFrancis83** [comment]
Yes it seems i managed to overcome the problem marking the whole button. But it has been quite difficult since I have also the problem of coma instead of dot.

**Comment by dickreuter** [comment]
Ok then maybe an option to switch from comma to dot might be more useful

From: Enrico ***@***.***>
Date: Thursday, 4 May 2023 at 18:05
To: dickreuter/Poker ***@***.***>
Cc: Nicolas Dickreuter ***@***.***>, Comment ***@***.***>
Subject: Re: [dickreuter/Poker] Bet button value and Raise button value located in two different areas. A distinct Bet button set up should be implemented (Issue #174)

Yes it seems i managed to overcome the problem marking the whole button. But it has been quite difficult since I have also the problem of coma instead of dot.
